pennylane/templates/subroutines/qdrift.py

```diff
@@ -144,6 +144,39 @@ def my_circ(time):
 
     """
 
+    @classmethod
```

Comment:
Potentially could be a good separate task to make all of these "with target operations" operators inherit from `SymbolicOp`. I think that would need to be a separate task, but it would. make a lot of these registrations a lot easier, since they would just inherit the behavior from `SymbolicOp`.

Reply:
I'll create a PR :)